src/gql/queries/get-my-hosts.ts

```diff
@@ -19,6 +19,10 @@ export const GET_MY_HOSTS = gql`
         value
         canBeModified
       }
+      volumes {
+        displayName
```

Comment:
Adding this query breaks the spawn host page when we hit the "unable to find volume error" 
![image](https://user-images.githubusercontent.com/4605522/94304512-aa17dd00-ff3d-11ea-8737-dce0c30c42b8.png)

Reply:
addressed in https://github.com/evergreen-ci/evergreen/pull/4040